packages/tailwind/src/tailwind-presets.ts

```diff
@@ -103,30 +95,58 @@ export const kaizenTailwindTheme: KaizenTailwindTheme = {
     "height-data-units-sm": `${defaultTheme.typography.dataSmallUnits.lineHeight}`,
   },
   letterSpacing: {
-    "spacing-normal": "normal",
+    "letter-spacing-normal": "normal",
   },
   // A mix of layout styles
   maxWidth: {
-    ...twDefaultTheme.maxWidth,
-    content: defaultTheme.layout.contentMaxWidth,
-    "content-with-side": defaultTheme.layout.contentMaxWidthWithSidebar,
+    "w-0": "0",
+    "w-25": "25%",
+    "w-50": "25%",
+    "w-75": "75%",
+    "w-100": "100%",
+    "w-auto": "auto",
+    "w-min": "min-content",
+    "w-max": "max-content",
+    "w-fit": "fit-content",
   },
   width: {
-    ...twDefaultTheme.width,
-    "content-side-margin": defaultTheme.layout.contentSideMargin,
+    "w-0": "0",
+    "w-25": "25%",
+    "w-50": "25%",
```

Comment:
```suggestion
    "w-50": "50%",
```
Since width and maxWidth seem to share the same values, it might be worth making an object?